changelog/8.6.0.yaml

```
@@ -0,0 +1,212 @@
+version: 8.6.0
+entries:
+    - kind: feature
+      summary: Only support elasticsearch as an output for the beta synthetics integration.
+      description: ""
+      component: synthetics-integration
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1491
+      issue: []
+      timestamp: 1665780486
+      file:
+        name: 1665780486-heartbeat-es-output-only.yaml
+        checksum: 361f993ba3220aef531f7949308e61d2cbf82e03
+    - kind: feature
+      summary: Upgrade node to 18.12.0
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1657
+      issue: []
+      timestamp: 1667243040
+      file:
+        name: 1667243040-Upgrade-node-to-18.12.0.yaml
+        checksum: 38b5473e0ee1165ad9c5d74ee969a9228c3fa68a
+    - kind: feature
+      summary: Add experimental support for running the elastic-agent-shipper
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1527
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/219
+      timestamp: 1667571017
+      file:
+        name: 1667571017-Add-support-for-running-the-elastic-agent-shipper.yaml
+        checksum: b2f5a0a8f9933c9ee624457116295aa674ef2a23
+    - kind: feature
+      summary: Add Common Expression Language input mapping to the Filebeat spec.
+      description: ""
+      component: spec
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1719
+      issue: []
+      timestamp: 1668483030
+      file:
+        name: 1668483030-common-expression-language.yaml
+        checksum: 82427fc514372c067e140837177c8d81131ed424
+    - kind: feature
+      summary: Capture stdout/stderr of all subprocesses and adjust default log level to info for all components.
```

```suggestion
      summary: Collect logs from sub-processes via stdout and stderr and write them to a single, unified Elastic Agent log file.
```

We should mention that all the logs are going into one file explicitly or we might get questions about it.